fabric/bin/lib/common_ledger.sh

```diff
@@ -48,6 +48,7 @@ define_common_vars() {
     : ${CHAN_ID:="mychannel"}
     : ${ERCC_ID:="ercc"}
     : ${ERCC_VERSION:="0"}
+    : ${SEQUENCE:="1"}
```

Comment:
for next teim, given that this is an sequence number for ercc, prefixing the var with ERCC_ as for the other variables would have been a bit more consistant